### ISSUE TITLE
cardano-chain tests use all the system's memory

### ISSUE THREAD

**Issue by dnadales** [input-output-hk/cardano-ledger-byron#212]
Steps to reproduce: clone the repository, initialize and update the submodules, and run:

```sh
stack test cardano-chain --ta --scenario=ContinuousIntegration
```

This uses all the memory available on my system (about 16GB) and consumes more than 30GB on CI.

**Comment by mdimjasevic** [comment]
I just made a PR #217 that should be a temporary work around.

**Comment by dnadales** [comment]
[This](https://github.com/input-output-hk/cardano-chain/blob/6f0cf2769a8c778f977df3e16a7f7e5be6e10434/test/test.hs#L31) is the validation test that causes the memory leak. The other validation test (`Test.Cardano.Chain.Txp.Validation.tests`) doesn't exhibit this problem.

**Comment by dnadales** [comment]
@redxaxder this was fixed by #230 wasn't it?

**Comment by dnadales** [comment]
Closing it since this seems to be solved. If not please re-open.
